README.md

```diff
@@ -0,0 +1,38 @@
+# enterprise-catalog plugin for [Tutor](https://docs.tutor.overhang.io)
+
+## Installation
+
+
+```sh
+pip install git+https://github.com/open-craft/tutor-contrib-enterprisecatalog
+```
+
+## Usage
+
+```sh
+# This plugin depends on discovery, ecommerce and mfe tutor plugins
+tutor plugins enable enterprise-catalog discovery ecommerce mfe
```

Comment:
Running this command results in the following error:
```python
tutor/palm via 🐍 v3.8.19 (tutor-palm) ➜ tutor plugins enable enterprise-catalog discovery ecommerce mfe
Error applying action: func=<function discover_package.<locals>.load at 0x7e29006d8160> contexts=['plugins']'
Traceback (most recent call last):
  File "~/.virtualenvs/tutor-palm/bin/tutor", line 8, in <module>
    sys.exit(main())
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/tutor/commands/cli.py", line 27, in main
    cli()  # pylint: disable=no-value-for-parameter
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/click/decorators.py", line 45, in new_func
    return f(get_current_context().obj, *args, **kwargs)
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/tutor/commands/plugins.py", line 140, in enable
    plugins.load(plugin)
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/tutor/plugins/__init__.py", line 98, in load
    hooks.Actions.PLUGIN_LOADED.do(name)
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/tutor/core/hooks/actions.py", line 116, in do
    self.do_from_context(None, *args, **kwargs)
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/tutor/core/hooks/actions.py", line 135, in do_from_context
    callback.do(
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/tutor/core/hooks/actions.py", line 36, in do
    self.func(*args, **kwargs)
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/tutor/plugins/v1.py", line 77, in load
    entrypoint.load()
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2496, in load
    return self.resolve()
  File "~/.virtualenvs/tutor-palm/lib/python3.8/site-packages/pkg_resources/__init__.py", line 2502, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "~/tutor/palm/plugins/tutor-contrib-enterprisecatalog/tutorenterprisecatalog/plugin.py", line 198, in <module>
    def _mount_repositories(mounts: list[tuple[str, str]], name: str) -> list[tuple[str, str]]:
TypeError: 'type' object is not subscriptable
```

I'm using the following versions:
```fish
➜ pip list |ag tutor
tutor                           16.1.8
tutor-android                   16.0.0
tutor-cairn                     16.0.3
tutor-contrib-enterprisecatalog 16.0.0      ~/tutor/palm/plugins/tutor-contrib-enterprisecatalog
tutor-discovery                 16.0.2
tutor-ecommerce                 16.0.4
tutor-forum                     16.0.2
tutor-license                   16.0.0
tutor-mfe                       16.1.3
tutor-minio                     16.0.4
tutor-notes                     16.0.2
tutor-webui                     16.0.1
tutor-xqueue                    16.0.2
```

I set up a clean Tutor environment with a custom `$TUTOR_ROOT` env variable to test this.

Reply:
@Agrendalath Actually, all latest tutor plugins seem to work on 3.9 and above versions only. I'll remove 3.5 to 3.8 versions from setup.py file. 
Can you please try with a new virtualenv with python 3.11?